src/graphics/program-lib/standard.js

```diff
@@ -1208,9 +1218,11 @@ pc.programlib.standard = {
             }
         }
 
+        var gloss_called = false;
```

Comment:
This variable naming is wrong. Should be camel case: i.e. `glossCalled`

Reply:
renamed to getGlossinessCalled